File: uri/1008/1008.cpp
#include <cstdio>

int main(){
    int a, b;
    float c;
    scanf("%d", &a);
    scanf("%d", &b);
    scanf("%f", &c);
    printf("NUMBER = %d\n", a);
    printf("SALARY = U$ %.2f\n", c * b);
    return 0;
}

File: solutions/uri/1008/1008.cs
using System;

class Solution {
    static void Main() {
        int a, b;
        double c;

        a = Convert.ToInt16(Console.ReadLine());
        b = Convert.ToInt16(Console.ReadLine());
        c = Convert.ToDouble(Console.ReadLine());

        Console.WriteLine("NUMBER = {0}", a);
        Console.WriteLine("SALARY = U$ {0:F2}", c * b);
    }
}
